integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -161,6 +161,12 @@ setup() {
 	docker run --rm --runtime=$RUNTIME -i $image bash -c "echo -e '#include<stdio.h>\nint main (void)\n{printf(\"Hello\");return 0;}' > demo.c && gcc demo.c -o demo && ./demo"
 }
 
+
+@test "[gradle] run a gradle container" {
+	image="gradle"
+	docker run --rm --runtime=$RUNTIME -i $image bash -c "echo -e 'public class CL{public static void main(String[]a){System.out.println(\"KataContainers\");}}' > CL.java"
```

Comment:
Shouldn't we try to compile this or is the intent of the test just to write the file and exit?

Reply:
@ganeshmaharaj changes applied